content/games/factions/routines.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Routines"
+description: "Rules for non-player Companies in Flagrant Factions"
+weight: 50
+---
+
+These procedures and guidelines apply to Companies not controlled by a player. Unless otherwise
+specified, Routine Companies follow the same rules as other companies. Where decisions are unclear,
+use judgement.
+
+## Personalities
+
+Routine companies have a Personality. Roll 1d6 or choose for each Routine Company:
+
+1. **Incompetent:** Follow default/random orders.
+1. **Arcane:** Prefer to use and acquire magic over mundanity.
+1. **Bold:** Chase objectives and move without concern for enemy action.
+1. **Cautious:** Steer clear of enemies unless odds are favorable.
+1. **Ferocious:** Destroy the enemy whenever possible.
+1. **Guarded:** Avoid attacking directly, prefer to hold and whittle.
+
+## Traits
+
+Where a trait would be triggered by player choice, roll 1d6: on 4+, the trait triggers.
+
+## Setup
+
+### Initiative
+
+Incompetent Companies automatically forfeit their initiative roll, counting as a 1 for all purposes.
+When a Routine Company wins the initiative roll-off:
```

Comment:
🤔 I think we should go with option 2 and retain the multiplayer stuff here. 

I'm chewing over the idea that all companies roll for initiative and add their results to see which side wins. This would favor the side that has more companies pretty heavily, which makes some sense to me (more captains means more skilled people prepared to seize initiative).

I think the rest of the text doesn't account well for n-sided skirmishes (3+ sides). I think nearly everything is written with an attacker/defender dichotomy. This might not be a bad time to address that. I'm leaning towards explicit support but open to leaving that to the reader to figure out.

Reply:
This would also mean clarifying which Company on a side goes first, which I think is also ambiguous right now.

Reply:
Okay so here's where I'm sitting with this, atm. I'm slightly wary of getting too "comprehensive" in the core rules. You're always at risk of beginning a recipe for bread with instructions for hybridizing your own strain of wheat. I'm into the idea of providing some kind of support, without getting to hysterical about it. The two options I like best are:

1. Just adding a little line/paragraph somewhere in the core rules like "Yo this is structured for attacker/defender, and assumes one company per side. We've tried to keep it a little open, but you'll have to do some legwork. I believe in you."
2. Add a little section to the core rules with some more explicit idea of how to handle the outside cases: What if >2 sides? What if >1 company on a side? This would be its own entire section, at the end of the thing, out of the way. It *might* involve tightening up the core to actually sit more squarely into "This is for two sides with one Company per side," but that's a bridge we can cross when we come to it.

Doing one of those things in the core rules would then give us some guidance as to how to handle things here in the auto rules. We'd basically mirror that same ethos from there to here, either keeping things as "This is attacker/defender and 1 Company each, figure it out if you want it different" or "That same thing, but here's a small section to account for n sides and n companies per side."

Reply:
I'm leaning very slightly towards 1. I think we already handled how the third-party non-player "companies" worked in the scenarios for some inspiration. Not covering it here gives us an excuse to chuck it in an open warfare supplement with specific scenarios for larger fights or w/e. Maybe that just belongs in the Eternal Hex which covers a multiplayer campaign where differently controlled companies can intercede.

Reply:
I [took a poke at addressing this in this document](https://github.com/FlagrantGarden/site/pull/52/files#r938106350), once we're happy with the wording we can probably pull it very nearly across. Or this could live only in the core rules or the frontmatter of the text when we're setting the context and just let people understand we meant the _whole_ text.

Reply:
I like this solution. I think it's the shortest path from here to there.